src/docfx/lib/HtmlSanitizer.cs

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using HtmlReaderWriter;
+using Markdig.Syntax;
+
+namespace Microsoft.Docs.Build
+{
+    internal class HtmlSanitizer
+    {
+        public static readonly Dictionary<string, HashSet<string>?> DefaultAllowedHtml = new(StringComparer.OrdinalIgnoreCase)
```

Comment:
why do we still need keep this allow list? how to keep them synced with the allowed list in PoolParty?
If we fail to get the allowlist from PoolParty, can we just failed the build because we think this is important from security perspective

Reply:
the allow list is default one of docfx and used in test case. It will be replaced by PoolParty data so build will fail when HTTP request to PoolParty failed.

Reply:
This is mostly for docfx public release and wont be the same as poolparty.